modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -4611,6 +4630,61 @@ public IgniteSnapshotFutureImpl(IgniteInternalFuture<Void> fut) {
         }
     }
 
+    /** Snapshot check operation status. */
+    public class CheckOperationStatus implements AutoCloseable {
```

Comment:
Public nested class. Let's create a separate file for it

Reply:
Fixed